macos/generate_wazuh_packages.sh

```diff
@@ -41,6 +43,11 @@ function clean_and_exit() {
     exit ${exit_code}
 }
 
+ctrl_c() {
```

Comment:
Here you should add `function` keyword before `ctrl_c`. Before macOS 10.15, `bash` is the default shell in macOS, but in macOS 10.15, `zsh` is the new default shell and it uses `bash` syntax. So, you should add `function` or remove any `function` keyword for consistency in this script.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/329/commits/45c5eb5ce3b01efdc2ac7be544d243b58e476832.